cnf-tests/testsuites/e2esuite/security/tuning.go

```diff
@@ -94,4 +94,35 @@ var _ = Describe("tuningcni", func() {
 			Expect(err).ToNot(HaveOccurred())
 		})
 	})
+
+	Context("tuningcni over bond", func() {
+		It("should be able to create pod with sysctls over bond",
+			func() {
+				macvlanNadName := "macvlan-nad"
+				bondNadName := "bond-nad"
+				sysctlValue := "1"
+				sysctls, err := networks.SysctlConfig(map[string]string{fmt.Sprintf(Sysctl, "IFNAME"): sysctlValue})
+				Expect(err).ToNot(HaveOccurred())
+
+				bondWithTuningNad, err := networks.NewNetworkAttachmentDefinitionBuilder(TestNamespace, bondNadName).WithBond("net3", "net1", "net2").WithStaticIpam("10.10.0.12").WithTuning(sysctls).Build()
+				Expect(err).ToNot(HaveOccurred())
+				err = client.Client.Create(context.Background(), bondWithTuningNad)
+				Expect(err).ToNot(HaveOccurred())
+
+				macVlandNad, err := networks.NewNetworkAttachmentDefinitionBuilder(TestNamespace, macvlanNadName).WithMacVlan().Build()
+				Expect(err).ToNot(HaveOccurred())
+				err = client.Client.Create(context.Background(), macVlandNad)
+				Expect(err).ToNot(HaveOccurred())
+
+				podDefinition := pods.DefineWithNetworks(TestNamespace, []string{fmt.Sprintf("%s/%s, %s/%s, %s/%s", TestNamespace, macvlanNadName, TestNamespace, macvlanNadName, TestNamespace, bondNadName)})
+				pod, err := client.Client.Pods(TestNamespace).Create(context.Background(), podDefinition, metav1.CreateOptions{})
+				Expect(err).ToNot(HaveOccurred())
+				err = pods.WaitForPhase(client.Client, pod, corev1.PodRunning, 1*time.Minute)
+				Expect(err).ToNot(HaveOccurred())
+				sysctlForInterface := fmt.Sprintf(Sysctl, "net1")
```

Comment:
How do we know that net1 is the bond interface?
Maybe it's worth inspecting the networkstatus annotation of the pod to make sure it's the case?

Reply:
I updated the code to have interface names specified explicitly in the networks annotation (instead of relying on multus names). They are also not named "net*", so no chance of accidental match.
Previously the code relied on how multus was naming the interfaces.
Hope that makes it more clear.